core/src/main/java/Main.java

```diff
@@ -25,6 +25,7 @@
 import edu.ucr.cs.riple.core.Annotator;
 import edu.ucr.cs.riple.core.Config;
 
+@SuppressWarnings("DefaultPackage")
```

Comment:
Why suppressing instead of making this `package edu.ucr.cs.riple.core;` and changing whatever command is used to run it or the jars metadata?

Reply:
@nimakarimipour ?

Reply:
Moved it to the `package edu.ucr.cs.riple.core`. d023ea6635a4652f59a420cf1fb8f449209099cc